src/PrestaShopBundle/Resources/config/services/core/localization.yml

```
@@ -1,30 +1,41 @@
 services:
   _defaults:
-    public: true
+    public: false
+    autowire: true
+
+  PrestaShop\PrestaShop\Core\Localization\Pack\Factory\LocalizationPackFactoryInterface: '@PrestaShop\PrestaShop\Core\Localization\Pack\Factory\LocalizationPackFactory'
+  PrestaShop\PrestaShop\Core\Localization\Pack\Factory\LocalizationPackFactory:
+  PrestaShop\PrestaShop\Core\Localization\Pack\Import\LocalizationPackImporter:
+    arguments:
+      $remoteLocalizationPackLoader: '@PrestaShop\PrestaShop\Core\Localization\Pack\Loader\RemoteLocalizationPackLoader'
+      $localLocalizationPackLoader: '@PrestaShop\PrestaShop\Core\Localization\Pack\Loader\LocalLocalizationPackLoader'
 
+  PrestaShop\PrestaShop\Core\Localization\Pack\Loader\LocalLocalizationPackLoader:
+  PrestaShop\PrestaShop\Core\Localization\Pack\Loader\RemoteLocalizationPackLoader:
+  PrestaShop\PrestaShop\Core\Localization\RTL\StyleSheetProcessorFactory:
```

I prefer having `~` as it seems to be more readable. But its just my preference, not requesting to change it :man_shrugging: 
```suggestion
  PrestaShop\PrestaShop\Core\Localization\Pack\Loader\LocalLocalizationPackLoader: ~
  PrestaShop\PrestaShop\Core\Localization\Pack\Loader\RemoteLocalizationPackLoader: ~
  PrestaShop\PrestaShop\Core\Localization\RTL\StyleSheetProcessorFactory: ~
```

---

I'll use service discovery in another PR as done in the form type PR, so this may not be needed right now. WDYT ?